### PR TITLE
LPS-177053 | master

### PR DESCRIPTION
**Pull request by kristinonias**
https://issues.liferay.com/browse/LPS-177053

### CODE REVIEW

**Comment by patriciajeanperez** [comment]
Just started reviewing :)

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
Thank you for also making that test easier to read! @kristinonias

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -3182,41 +3182,49 @@ definition {
...
-		WorkflowMetrics.goToWorkflowAllItems(workflowProcessName = "Single Approver");
+			JSONBlog.addEntry(
+				entryContent = "Test Content",
+				entryTitle = "Test Blog");
+		}
 
-		Click(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+		task ("Then the SLA status should be related to the moment the pause timer triggered") {
+
+			WorkflowMetrics.goToWorkflowAllItems(workflowProcessName = "Single Approver");
```
Comment (timpak) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase]: Looks like there's an unnecessary space here, which is why SF is failing.

Comment (timpak) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase]: @kristinonias

**Comment by patriciajeanperez** [comment]
See https://github.com/liferay-bpm-qa/liferay-portal/pull/678

**Comment by patriciajeanperez** [comment]
ci:close

**Pull request closed without merging**